components/common/privateCDN.ts

```diff
@@ -3,6 +3,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
+const DEFAULT_MIMETYPE = 'image/jpg'
```

Comment:
How flexible can the code be here? Can we have the client guess the mimetype based on the extension (one of: `jpg`, `png`, `webp`, `avif`)?

I'm asking because [we're talking](https://github.com/brave/security/issues/616#issuecomment-943911667) about forcing a conversion away from the input format in the case of ads.